docs/docs/technology/restaking/roadmap.md

```diff
@@ -0,0 +1,13 @@
+# Roadmap
+
+The rollout plan for Picasso Generalized restaking will proceed as follows:
+
+1. Initial implementation of the [Restaking Vaults on Solana](../restaking/vaults.md).
+2. Launch of the first [AVS for Solana IBC](../restaking/sol-ibc-avs.md).
+3. Begin onboarding AVSs onto the Solana Restaking layer.
```

Comment:
We need the generalized restaking layer to be available on the cosmos chain earlier